lucille-core/src/test/java/com/kmwllc/lucille/core/RunnerTest.java

```diff
@@ -677,15 +681,38 @@ public void testConnectorWithUnrecognizedPipeline() throws Exception {
     RunResult result =
         Runner.run(ConfigFactory.load("RunnerTest/pipelineNotFound.conf"), Runner.RunType.TEST);
     assertFalse(result.getStatus());
+
+    Collection<Thread> nonSystemThreadsAfter =
+        ThreadUtils.findThreads(t -> !ThreadUtils.getSystemThreadGroup().equals(t.getThreadGroup()));
+
+    for (Thread thread : nonSystemThreadsAfter) {
+      log.info("THREAD testConnectorWithUnrecognizedPipeline AFTER: {}", thread.getName());
+      if (thread.getName().startsWith(ThreadNameUtils.THREAD_NAME_PREFIX)) {
```

Comment:
same comment re: adding ThreadNameUtils.isLucilleThread()

Reply:
resolved in [2044c9f](https://github.com/kmwtechnology/lucille/pull/172/commits/2044c9f47eab57317e613780caed80c0de5958fd)